src/common/fs/FileUtils.cpp

```diff
@@ -16,62 +16,60 @@ static const int32_t kMaxPathLen = 1024;
 
 namespace detail {
 
-bool removeDir(const char* path, bool recursively) {
-    if (recursively) {
```

Comment:
Good catch!

But I notice that `removeDir` behaves differently with the standard `rmdir` API on what's recursive. This might be counter intuitive, at least to me. But let us discuss this later.